content/go/server.go.md

```diff
@@ -558,7 +558,7 @@ func cancelServer(serverId int) {
 	mask := "id;hostname;billingItem;hourlyBillingFlag"
 	server := getServerIntance(serverId, mask)
 
-	// If server is billed hourly it will be cancelled immediately, for monthly server
+	// If server is billed hourly it will be canceled immediately, for monthly server
```

Comment:
cancelled

Reply:
Since both options seem valid, I keep the current one.